contracts/governance/TimelockController.sol

```diff
@@ -199,6 +177,22 @@ contract TimelockController is AccessControl, ERC721Holder, ERC1155Holder {
         return _timestamps[id];
     }
 
+    /**
+     * @dev Returns operation state.
+     */
+    function getOperationState(bytes32 id) public view virtual returns (OperationState) {
+        uint timestamp = getTimestamp(id);
```

Comment:
We favor uint256 over uint to achieve maximum clarity. Some people can be confused about what a uint is.
```suggestion
        uint256 timestamp = getTimestamp(id);
```

Reply:
Done